hbase-http/src/main/java/org/apache/hadoop/hbase/http/log/LogLevel.java

```diff
@@ -345,6 +347,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         out.println(MARKER
             + "Log Class: <b>" + log.getClass().getName() +"</b><br />");
         if (level != null) {
+          if (!isLogLevelChangeAllowed(logName, readOnlyLogLevels)) {
+            response.sendError(HttpServletResponse.SC_PRECONDITION_FAILED,
```

Comment:
After Jetty 9.4.21, sendError() no longer allows a custom message.
You can use setStatus() instead. See https://github.com/apache/hadoop/blob/trunk/hadoop-common-project/hadoop-auth/src/main/java/org/apache/hadoop/security/authentication/server/AuthenticationFilter.java#L622 for reference.

Reply:
Thanks for the pointer. It looks like hbase uses a shaded version of netty, and further up in this class uses `sendError` to send an error response. I could change this if I get another vote from an hbase committer, but I probably should change both in that case. Seems out of scope for this jira, but can do it if we think I should.

Reply:
@jojochuang I looked more closely, and there are at least 6 servlets that use the current `sendError` behavior, as well as a couple filters. It seems like we might want to leave this alone here and instead create a jira to update all usages so that they are future proof. Or we could fix those issues as part of a future "upgrade netty" jira.

Does that sound reasonable?